node/network/dispute-distribution/src/receiver/mod.rs

```diff
@@ -105,8 +103,15 @@ pub struct DisputesReceiver<Sender, AD> {
 	/// Rate limiting queue for each peer (only authorities).
 	peer_queues: PeerQueues,
 
-	/// Delay timer for establishing the rate limit.
-	rate_limit: Fuse<Delay>,
+	/// Time to check on batches whether they are ready for import.
+	check_batches_waker: Option<Fuse<Delay>>,
+
+	/// Currently active batches of imports per candidate.
+	///
+	/// We use an `IndexMap` here as the order of insertion is important.
```

Comment:
Here the comment says we use an IndexMap, but it appears that the code is using an ordinary HashMap. Possibly an outdated comment?

Reply:
Indeed, thanks!